mlflow/R/mlflow/DESCRIPTION

```diff
@@ -40,17 +40,20 @@ Imports:
     xml2,
     yaml,
     zeallot
-Suggests: 
+Suggests:
   covr,
   carrier,
+  dplyr,
```

Comment:
I vote against adding a suggested dplyr dependency. The 3 calls you use dplyr for in the tests are in MHO not worth the extra dependency (haven't checked if dplyr was in the recursive dependency tree already but still), in particular these can be achieved easily with base R.